test_onnx.py

```
@@ -0,0 +1,186 @@
+        return False
+
+    # Mock API
+    mock_api = MagicMock()
+    mock_api.list_tasks.return_value = [
+        {"id": 1, "task_type": "onnx", "title": "Test", "data": {}}
+    ]
+    mock_api.mark_assignment_as_failed = MagicMock()
+    mock_fedledger.return_value = mock_api
+
+    # Mock HTTP requests for CSV data (use real HuggingFace download for model)
+    def mock_get_side_effect(url):
```

Ruff (0.14.3)

Unused function argument: `url`

(ARG001)